plugins/GrapesJsBuilderBundle/Assets/library/js/builder.js

```diff
@@ -70,11 +70,42 @@ function setThemeHtml(theme) {
   });
 }
 
+/**
+ * The builder button to launch GrapesJS will be disabled when the code mode theme is selected
+ *
+ * @param theme
+ */
+function switchBuilderButton(theme) {
```

Comment:
I don't know If GrapesJsBuilderBundle is right place to put it.
We need cover it for emails and pages in general not related what builder is enabled.
I understand legacy builder is removed in M5 but If we push this now, we need rewrite it in M5 again.